lib/brakeman/util.rb

```diff
@@ -124,6 +124,18 @@ def hash_access hash, key
     nil
   end
 
+  def call_has_param call, key
```

Comment:
Damn. Sorry, such a newbie mistake. I am almost there. That's the method and almost all tests are working. Except by this one: `Rails4Tests#test_zero_errors [test/test.rb:89]:`

The stack trace is this: 

```
Unexpected warning found: [{:error=>"Expected call or attrasgn or safe_call or safe_attrasgn or super or zsuper or result but given s(:const, :User)", :backtrace=>["/home/user/brakeman/lib/brakeman/checks/check_redirect.rb:124:in `call_has_param'", "/home/user/brakeman/lib/brakeman/checks/check_redirect.rb:119:in `use_unsafe_hash_method?'", "/home/user/brakeman/lib/brakeman/checks/check_redirect.rb:111:in `only_path?'", "/home/user/brakeman/lib/brakeman/checks/check_redirect.rb:38:in `process_result'", "/home/user/brakeman/lib/brakeman/checks/check_redirect.rb:27:in `block in run_check'", "/home/user/brakeman/lib/brakeman/checks/check_redirect.rb:26:in `each'", "/home/user/brakeman/lib/brakeman/checks/check_redirect.rb:26:in `run_check'", "/home/user/brakeman/lib/brakeman/checks.rb:195:in `run_a_check'", "/home/user/brakeman/lib/brakeman/checks.rb:131:in `block (2 levels) in actually_run_checks'"]}].
Expected: 0
  Actual: 1
```

Not sure what this really means, but the error first line of error is `method = target.method` from the `call_has_params` method. Don't know how to fix that. 

Thanks for your guidance, btw :)